Dockerfile

```diff
@@ -2,8 +2,15 @@ FROM fedora:24
 ENV LANG C
 
 RUN echo "deltarpm=0" >> /etc/dnf/dnf.conf
+RUN curl https://copr.fedorainfracloud.org/coprs/ignatenkobrain/behave-enhanced/repo/fedora-24/ignatenkobrain-behave-enhanced-fedora-24.repo -o /etc/yum.repos.d/behave-enhanced.repo
```

Comment:
for a while it's good idea because it's still not pushed into repos. Once it will hit repos, I will change it once more.